src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,133 @@
+package wolox.training.models;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String genre;
+
+    @Column(nullable = false, unique = true)
```

Comment:
Use the unique attribute when the attribute is really unique.
In this case only the isbn attribute is unique.

Reply:
it is true